<launch>
    <arg name="yaml_file" default="tdoa_params"/>

    <node name="tdoa_v2" pkg="task2" type="tdoa_v2.py" output="screen" clear_params="true" required="true">
        <rosparam command="load" file="$(find task2)/config/$(arg yaml_file).yaml" />
    </node>
</launch>
